index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header>
+      <a id="logo">Farida's Logo</a>
+      <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
+    </header>
```

Comment:
- [ ] Kindly decrease the top spacing between the header and the top of the page to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/164271778-7b960c8c-4686-49dc-a556-b2915a1647a0.png) | ![image](https://user-images.githubusercontent.com/24703171/164271979-3fd8d97f-0edd-480e-9814-9810370b5317.png) |